### DISCUSSION TITLE
Make a hyprdots wiki??

### DISCUSSION THREAD

**Discussion by Enigma1309** [prasanthrangan/hyprdots#1046] [Ideas]
I've really liked this dots and have been using for about 9-10 months now (June i guess)
I've been thinking about making a static wiki page for our repo , like :- https://end-4.github.io/dots-hyprland-wiki/en/

any thought on this?

**Comment by prasanthrangan** [comment]
This looks so cool, I'd like to make one for this repo.
But I've no idea how to make it :)

**Reply by Enigma1309** [reply]
we can start with something simple like retype and hugo they are markdown websites, and are easy to maintain, or can go with [solidcli](https://solid-cli.netlify.app/) (aniyomi,mihon,fmhy, and a lot of sites use this)

I'm willing to help, rn my HS exams are going on but still i can spare some time on this project too

**Reply by prasanthrangan** [reply]
Thank you for the support! `hugo` looks simple and easy at a glance, so happy to go with it.
No worries, take your time and all the best for your exams.
Let me know if you need any input from me, I'll try to document as much as I can... I too have an exam this month :)

**Reply by Enigma1309** [reply]
That sounds nice, also please make a separate repo for Hyprdots wiki so that we can also contribute to it.
i would suggest you to use [this](https://lotusdocs.dev/docs/) theme for hugo since hyprland wiki is probably gonna use this in future , anyways the choice is yours .
btw thanks i wish you best of luck  for your exams too.
